docs/en_us/developers/README.rst

```diff
@@ -0,0 +1,45 @@
+######################
+EdX Developer's Guide
+######################
+
+We have moved the edX Developer's Guide to the `edx-documentation`_ repository as of January 13, 2015.
+
+By moving documentation to its own repository, we will be better able to
+develop workflows, manage versioning, create translations, and automate
+testing, without complicating ongoing development of the edX Platform.
+
+.. _edx_documentation: https://github.com/edx/edx-documentation
+
+******************************
+EdX Platform Docstrings
+******************************
+
+We are in the process of creating a new project to publish docstrings for the
+edX Platform. Docstrings will be included for CMS, LMS, and Common modules.
+
+This project will remain in the edX Platform repository, in the ''docs/en-us''
+directory.
+
+******************************
+View Published Documentation
+******************************
+
+EdX documentation is published through Read the Docs. Links to all published
+documentation are available through `docs.edx.org`_.
+
+.. _docs.edx.org: http://docs.edx.org
+
+******************************
+Submit Documentation Issues
+******************************
+
+We welcome input from the community on any documentation issues.  You can
+submit issues to the Documentation project in the `Open edX JIRA board`_.
+
+You will need to `create a free JIRA account`_ account before you can submit your first
+ticket.
+
+.. _create a free JIRA account: https://openedx.atlassian.net/admin/users/sign-up
+.. _Open edX JIRA board: https://openedx.atlassian.net
+
+You can also email docs@edx.org.
```

Comment:
becasue of the @ sign, this may not be formatted as expected in the HTML output.